js/src/edit-block/components/editor.js

```diff
@@ -126,70 +128,72 @@ const Editor = ( { onError, postId, postType, settings } ) => {
 				}
 				<BrowserURL />
 				<UnsavedChangesWarning />
-				<EditorProvider
-					post={ post }
-					settings={ settings }
-				>
-					<ErrorBoundary onError={ onError }>
-						<Header editorMode={ editorMode } setEditorMode={ setEditorMode } />
-						<EditorNotices />
-						<div className="gcb-editor flex w-full h-0 flex-grow">
-							<Main editorMode={ editorMode }>
-								<LocationButtons
-									currentLocation={ currentLocation }
-									editorMode={ editorMode }
-									setCurrentLocation={ setCurrentLocation }
-								/>
-								{ EDITOR_PREVIEW_EDITING_MODE === editorMode && block && block.fields
-									? <EditorPreview />
-									: null
-								}
-								{ BUILDER_EDITING_MODE === editorMode
-									? (
-										<FieldsGrid
-											currentLocation={ currentLocation }
-											selectedField={ selectedField }
-											setIsNewField={ setIsNewField }
-											setPanelDisplaying={ setPanelDisplaying }
-											setSelectedField={ setSelectedField }
-										/>
-									) : null
-								}
-								{ FRONT_END_PREVIEW_EDITING_MODE === editorMode
-									? <FrontEndPreview />
-									: null
-								}
-								{ TEMPLATE_EDITOR_EDITING_MODE === editorMode
-									? <TemplateEditor />
-									: null
-								}
-							</Main>
-							<Side
-								panelDisplaying={ panelDisplaying }
-								setPanelDisplaying={ setPanelDisplaying }
-							>
-								{
-									BLOCK_PANEL === panelDisplaying
-										? <BlockPanel />
+				<SlotFillProvider>
```

Comment:
This file isn't changed as much as it seems.

`<SlotFillProvider>` is new, and its children are indented one more leve.

Reply:
There aren't many other changes, other than passing some more props.